### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by CodeWriterWriter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
-# seneca-entity
-Entity plugin for seneca
+![Seneca](http://senecajs.org/files/assets/seneca-logo.png)
+> A Node.js toolkit for Micro-Service Architectures
+
+# seneca
```
Comment (mcdonnelldean) [inline README.md]: seneca-entity

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+[![npm version][npm-badge]][npm-url]
+[![Build Status][travis-badge]][travis-url]
+
+## About Seneca Entity
```
Comment (mcdonnelldean) [inline README.md]: Header can go

File: README.md [hunk 3]
```diff
@@ -1,2 +1,38 @@
...
+_Seneca Entity_ is an entity plugin for Seneca
+
+## Install
+To install, simply use npm.
+
+```
+npm install seneca-entity
+```
+
```
Comment (mcdonnelldean) [inline README.md]: Add quick example,

```js
require('seneca')()
  .use('entity')
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,38 @@
...
+## Test
+To run tests, simply use npm:
+
+```
+npm run test
+```
+## Contributors
```
Comment (mcdonnelldean) [inline README.md]: Where did this table come from? Is it used elsewhere?

Comment (CodeWriterWriter) [inline README.md]: Got the table format from the seneca-in-practice readme

Comment (mcdonnelldean) [inline README.md]: Yeah, don't use this, it should look closer to https://github.com/senecajs/seneca-level-store without the annotated or data specifc sections.

**Pull request merged**